sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/EncryptionKeyWrapMetadata.java

```diff
@@ -38,17 +38,15 @@ public EncryptionKeyWrapMetadata(EncryptionKeyWrapMetadata source) {
     /**
      * Creates a new instance of key wrap metadata based on an existing instance.
      *
+     * @param type Type of the metadata.
      * @param name Name of the metadata.
      * @param value Value of the metadata.
      */
     @Beta(value = Beta.SinceVersion.V4_14_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
-    public EncryptionKeyWrapMetadata(String name, String value) {
-        this("custom", name, value);
-    }
-
-    private EncryptionKeyWrapMetadata(String type, String name, String value) {
+    public EncryptionKeyWrapMetadata(String type, String name, String value) {
         Preconditions.checkNotNull(type, "type is null");
         Preconditions.checkNotNull(value, "value is null");
+        Preconditions.checkNotNull(name, "value is null");
```

Comment:
Incorrect message here - should be `name is null`

Reply:
corrected it